modules/configuring-your-ide.adoc

```diff
@@ -0,0 +1,174 @@
+
+For more information on code-server, see link:https://github.com/coder/code-server[code-server in GitHub].
+
+=== Incorporating DBMS clients using extensions
+
+.Prerequisites
+
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have created a data science project that has a code-server workbench.
+
+.Procedure
+
+. From the {productname-short} dashboard, click *Data Science Projects*.
++
+The *Data science projects* page opens.
+. Click the name of the project containing the code-server workbench you want to start.
++
+The *Details* page for the project opens.
+. Click the toggle in the *Status* column for the relevant workbench to start a workbench that is not running.
++
+The status of the workbench that you started changes from *Stopped* to *Running*. 
+. After the workbench has started, click *Open* to open the workbench notebook.
+. In the Activity Bar, click the Extensions icon. (image:images/codeserver-extensions-icon.png[The Extensions icon])
+. Search for the name of the extension you want to install. 
+. Click *Install* to add the extension to your code-server environment.
++ 
+The extension you installed appears in the *Browser - Installed* list on the *Extensions* panel.
+
+==== Extensions
+
+See link:https://marketplace.visualstudio.com/vscode[Visual Studio Marketplace] for available third-party extensions that you can consider installing.
+
+=== Incorporating DBMS clients using a custom image
+
+Custom notebook images are useful when working with containerized environments, especially when you need specific libraries, OS packages, or applications that are not available in base images. Follow these steps to create a custom notebook image with Database Management System (DBMS) clients for MongoDB, MSSQL, and PostgreSQL using a Dockerfile.
+
+.Prerequisites
+
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have created a data science project that has a code-server workbench.
+* You have installed Docker on your system.
+
+.Procedure
+
+. Clone the Open Data Hub *notebooks* repository:
```

Comment:
Around the clone step?

The original link provided for this one was an opendatahub-io one - should this now use the attribute because the link's been updated to  git@github.com:red-hat-data-services/notebooks.git?

Reply:
Removing the Custom image section for now.